s3auth-relay/src/main/java/com/s3auth/relay/HttpFacade.java

```
@@ -256,6 +259,9 @@ public void run() {
             } catch (final InterruptedException ex) {
                 Thread.currentThread().interrupt();
                 Logger.warn(this, "%s", ex);
+            // @checkstyle IllegalCatch (1 line)
+            } catch (final Throwable ex) {
+                Logger.error(this, "%s", ExceptionUtils.getStackTrace(ex));
```

swallowing the exception? is it intentionally or by mistake?